packages/kotti-dropdown-button/KtDropdownButton.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<div class="kt-dropdown-button">
+		<KtPopover v-bind="$attrs" :placement="placement">
+			<KtButtonGroup>
+				<KtButton class="kt-dropdown-button__button" v-bind="$attrs">
+					<slot>Dropdown Text</slot>
+				</KtButton>
+				<KtButton
+					class="kt-dropdown-button__icon"
+					v-bind="$attrs"
+					:icon="dropdownIcon"
+				>
+					<i class="yoco" v-text="dropdownIcon" />
+				</KtButton>
+			</KtButtonGroup>
+			<div slot="content">
+				<slot name="KtDropdownMenus">Dropdown Menu</slot>
```

Comment:
Why name the slot, why not just have it be the default slot

Reply:
I copied the behavior of the "classic" Dropdown. I don't think we should change it here if we don't change it in the `KtDoropdown` component as well.

Reply:
Slot name is very redundant. Just call it `menu` IMO.